airbyte-integrations/connectors/source-pipedrive/source_pipedrive/spec.json

```diff
@@ -12,6 +12,24 @@
         "title": "Authentication Type",
         "description": "Choose one of the possible authorization method",
         "oneOf": [
+          {
+            "type": "object",
+            "title": "API Key Authentication",
+            "required": ["auth_type", "api_token"],
+            "properties": {
+              "auth_type": {
+                "type": "string",
+                "const": "Token",
+                "order": 0
+              },
+              "api_token": {
+                "title": "API Token",
+                "type": "string",
+                "description": "The Pipedrive API Token.",
+                "airbyte_secret": true
+              }
+            }
+          },
           {
             "title": "Sign in via Pipedrive (OAuth)",
             "type": "object",
```

Comment:
Shouldn't this be removed as well?

Reply:
I have removed "Sign in via Pipedrive (OAuth)"